infer/bin/jwlib.py

```diff
@@ -31,7 +31,7 @@ def run(self):
         if self.args.version:
             return subprocess.call(['javac'] + self.original_arguments)
         else:
-            javac_cmd = ['javac', '-verbose', '-g'] + self.original_arguments
+            javac_cmd = ['javac', '-verbose', '-g', '-J-Duser.language=en'] + self.original_arguments
```

Comment:
I tried this with:

     `infer -- javac -verbose -J-Duser.language=ja Hello.java`

but it did not work for me because the command line then becomes:

  `['javac', '-verbose', '-g', '-J-Duser.language=en', '-verbose', '-J-Duser.language=ja', 'infer/bizarre/Test.java']`

So the second language option `'-J-Duser.language=ja'` is overriding the first one `'-J-Duser.language=en'`.

To make this work, you will need to add the local information at the end of the command line:

            javac_cmd = ['javac', '-verbose', '-g'] + self.original_arguments
            # force the verbose output to be in English
            javac_cmd.append('-J-Duser.language=en')

Will you be able to update the pull request to move the language option at the end ?

Thanks for this. It is good to make Infer more robust here because parsing the verbose output of javac is unfortunately a little bit fragile.